modules/puzzles/module.py

```diff
@@ -52,3 +53,40 @@ def buildings_with(self, symbols):
         results = util.buildings_containing_symbols(symbols)
         self.respond('Er zijn %i gebouwen die de symbolen \'%s\' bevatten' % (len(results), symbols))
         self.respond(', '.join([str(x) for x in results]))
+
+
+class Words(TelegramModule):
+    @command
+    def text_to_morse(self, *words):
+        """
+        Zet een aantal woorden om naar morse
+        """
+        try:
+            self.respond('\n'.join([' '.join(CODE[i.upper()] for i in word)
+                                   for word in words]))
+        except KeyError as e:
+            self.respond('Character %s cannot be parsed to morse, only characters and digits can'
```

Comment:
Maybe its a good idea to still provide a best effort convertion and replace the unknown words with a question mark or something. That way a single typo wont ruin the whole thing

Reply:
Did it, on unknown letters it gives a question mark